ret-cli/src/main/kotlin/io/rabobank/ret/plugins/PluginManager.kt

```diff
@@ -0,0 +1,28 @@
+package io.rabobank.ret.plugins
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import jakarta.enterprise.context.ApplicationScoped
+
+private val repoConfig = """
+    {
+      "name": "ret official",
+      "url": "https://raw.githubusercontent.com/stevenschenk/ret-repository/main/plugins.json"
```

Comment:
I know it doesn't exist yet, but let's assume this would be the real repo instead of yours?